src/app/pages/send/send-crypto-asset-preview/send-crypto-asset-preview.tsx

```diff
@@ -0,0 +1,104 @@
+import { useCallback, useEffect, useState } from 'react';
+import toast from 'react-hot-toast';
+import { useLocation, useNavigate } from 'react-router-dom';
+
+import { StacksTransaction, deserializeTransaction } from '@stacks/transactions';
+import get from 'lodash.get';
+
+import { logger } from '@shared/logger';
+import { RouteUrls } from '@shared/route-urls';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { useHomeTabs } from '@app/common/hooks/use-home-tabs';
+import { LoadingKeys } from '@app/common/hooks/use-loading';
+import { useRouteHeader } from '@app/common/hooks/use-route-header';
+import { useSubmitTransactionCallback } from '@app/common/hooks/use-submit-stx-transaction';
+import { Header } from '@app/components/header';
+import { useSignTransactionSoftwareWallet } from '@app/store/transactions/transaction.hooks';
+
+import { SendCryptoAssetButton } from './components/send-crypto-asset-button';
+import { SendCryptoAssetDetails } from './components/send-crypto-asset-details';
+import { SendCryptoAssetPreviewLayout } from './components/send-crypto-asset-preview.layout';
+
+// TODO: Handle bitcoin transactions
+export function SendCryptoAssetPreview() {
+  const [unsignedTransaction, setUnsignedTransaction] = useState<string | null>(null);
+  const { setActiveTabActivity } = useHomeTabs();
+  const signSoftwareWalletTx = useSignTransactionSoftwareWallet();
+  const analytics = useAnalytics();
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  useRouteHeader(<Header hideActions onClose={() => navigate(-1)} title="You'll send" />);
+
+  useEffect(() => {
+    const tx = get(location.state, 'tx');
+    if (tx) setUnsignedTransaction(tx);
+    void analytics.track('view_transaction_signing');
+
+    return () => setUnsignedTransaction(null);
+  }, [analytics, location.state]);
```

Comment:
Could be its own hook for a nicer api? Returning all the values that are lated pulled?

Also, think analytics isn't necessary as this is it's own route anyway?